build.sh

```diff
@@ -24,48 +24,32 @@ if [ -z "$HOME" ]; then
     mkdir -p $HOME
 fi
 
+args=( "$@" )
+
 while [[ $# > 0 ]]; do
     lowerI="$(echo $1 | awk '{print tolower($0)}')"
     case $lowerI in
-        -c|--configuration)
-            export CONFIGURATION=$2
-            shift
-            ;;
-        --nopackage)
-            export DOTNET_BUILD_SKIP_PACKAGING=1
-            ;;
         --docker)
             export BUILD_IN_DOCKER=1
             export DOCKER_IMAGENAME=$2
+            # remove docker args
+            args=( "${args[@]/$1}" )
+            args=( "${args[@]/$2}" )
```

Comment:
A caveat- It just removes the string. There is still an empty element in the array.